### ISSUE TITLE
Search function in Events tab does not search event names.

### ISSUE THREAD

**Issue by Silver-Streak** [4ian/GDevelop#1215]
## Describe the bug
When searching (Magnifying Glass) in the Events editor, if you search for the text of an event, no results are found. 

e.g. If I want to search for all "Stop Object" events, searching for "Stop" returns 0 results.

## To Reproduce

1. Add a "Stop Object" event to your event list.
2. Click the Search/Magnifying Glass icon.
3. Search for **Stop**. 
4. Click **Search**. Case insensitive and filters do not matter. No results will be found.

## Other details
OS: Windows 10 Pro 64 bit.
GDevelop Version: 5.0.0-beta79
Screenshots attached
![No results](https://user-images.githubusercontent.com/24237149/63657976-83044000-c76c-11e9-8218-4a21653caa73.png)

**Comment by 4ian** [comment]
Indeed, the search is for now only done on the parameters, not the full text of the action/condition. Might be worth updating this.

**Comment by keshavg2** [comment]
Hi @4ian How I proceeding with this issue?

**Comment by 4ian** [comment]
Might require a bit of work because you would need to generate the sentence of the action or condition (using `InstructionSentenceFormatter::Translate` from `InstructionSentenceFormatter`). Do this in the search, which is done in `EventsRefactorer` (`EventsRefactorer::SearchInEvents`, `SearchStringInActions`...).
You'll need the metadata of the instruction, which can be obtained using `gd::MetadataProvider`.

**Comment by tristanbob** [comment]
This may not be the right time to change this, but it is a related topic: 

**Increase the visibility of highlighted events.**  
(We can barely see that skinny light blue line on the left side.)

Otherwise, thanks for adding this feature!
